nuancier/templates/contribute_index.html

```diff
@@ -170,7 +170,31 @@
             {{ election.election_name }} - {{ election.election_year }}
         </a>
     </li>
-    {% endfor %}
+    <table>
+        <tr>
+            <th>Name</th>
+            <th>Year</th>
+            <th>Submissions</th>
+        </tr>
+        <tr>
+            <td>
+                <a href="{{ url_for('election', election_id=election.id) }}">
+                    {{ election.election_name }}
+                </a>
+            <td>{{ election.election_year }}</td>
+            <td>
+                {% if election.submission_open %}
+                <img src="{{ url_for('static', filename='Approved.png') }}"
+                    alt="Submission open"/>
+                {% else %}
+                <img src="{{ url_for('static', filename='Denied.png') }}"
+                    alt="Submission closed"/>
+                {% endif %}
+                {{ election.submission_date_start }}
+            </td>
+        </tr>
+        {% endfor %}
```

Comment:
I think the placement of this ``{% endfor %}`` might be problematic.

If there are 5 elections, then this code will open 5 ``<table>`` tags, but only close once at the end of the loop.

Do you see what I mean @farhaanbukhsh?

Perhaps the ``{% endfor %}`` should be moved down a few lines?